### PR TITLE
Add problems cards

### PR DESCRIPTION
**Pull request by spuigmitjadxc**

### CODE REVIEW

File: app/cells/decidim/problems/problem_m/show.erb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<div class="card card--problem <%= resource_state %> muted">
+    <div class="card__content">
+        <div class="card__header">
+            <%= link_to problem_path, class: "card__link" do %>
+                <div class="card__title">
+                    <%= resource_title %>
+                </div>
+            <% end %>
+        </div>
+        <%#= cell("decidim/coauthorships", problem, has_actions: false, context: { current_user: current_user }) %>
```
Comment (tramuntanal) [inline app/cells/decidim/problems/problem_m/show.erb:10]: remove

File: app/cells/decidim/problems/problem_m/show.erb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <div class="card__text--paragraph">
+                <span class="card__text--paragraph--challenge"><%= t("challenge", scope: 'decidim.problems') %>: </span>
+                <%= link_to challenge_path, class: "card__link card__text--paragraph--challenge--title" do %>
+                    <div><%= resource_challenge_title%></div>
+                <% end %>
+            </div>
+        </div>
+        <div class="card__text">
+            <div class="card__text--paragraph">
+                <span class="card__text--paragraph--estat"><%= t(resource_state, scope: 'decidim.problems.states') %></span>
```
Comment (tramuntanal) [inline app/cells/decidim/problems/problem_m/show.erb:21]: This should be rendered with all letters upcase, in challenges I got it rendered as normal text, but should be as follows in both challenges and problems:
imatge

Comment (tramuntanal) [reply app/cells/decidim/problems/problem_m/show.erb:21]: Also use always English in the code. Rename `card__text--paragraph--estat` please

File: app/views/decidim/problems/problems/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+              </div>
+              <img alt="Logo ODS <%= @sdg_index %>"
+                src="/assets/decidim/sdgs/ods-<%= @sdg_index %>.svg" class="problem--view">
+          </div>
+        </div>
+        </div>
+
+      <%= resource_reference(current_participatory_space) %>
+      <%= render partial: "decidim/shared/share_modal" %>
+      <!-- embed_modal_for participatory_process_participatory_process_widget_url(current_participatory_space, format: :js)  -->
```
Comment (tramuntanal) [inline app/views/decidim/problems/problems/show.html.erb:65]: remove

File: config/locales/ca.yml [hunk 4]
```diff
@@ -971,6 +973,9 @@ ca:
             success: Problema actualitzat correctament
         titles:
           problems_list: Llistat de problemes
+      show:
+        proposed_solutions: Solucions proposades
+        return_to_list: Tornar al llistat
```
Comment (tramuntanal) [inline config/locales/ca.yml:978]:
```suggestion
        return_to_list: Torna al llistat
```

File: spec/cells/decidim/problems/problem_cell_spec.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+module Decidim::Problems
+  describe ProblemCell, type: :cell do
+    controller Decidim::Problems::ProblemsController
+
+    let!(:problem) { create(:problem) }
+
+    context "when rendering" do
+      it "renders the card" do
+        html = cell("decidim/problems/problem", problem).call
+        expect(html).to have_css(".card--problem")
```
Comment (tramuntanal) [inline spec/cells/decidim/problems/problem_cell_spec.rb:14]: Can we also check the contents of the problem are rendered?

File: spec/cells/decidim/problems/problem_m_cell_spec.rb [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    let(:model) { problem }
+    let(:cell_html) { cell("decidim/problems/problem_m", problem, context: { show_space: show_space }).call }
+
+    it_behaves_like "has space in m-cell"
+
+    context "when rendering" do
+      let(:show_space) { false }
+
+      it "renders the card" do
+        expect(cell_html).to have_css(".card--problem")
```
Comment (tramuntanal) [inline spec/cells/decidim/problems/problem_m_cell_spec.rb:19]: Can we also check the contents of the problem are rendered?

**Review by tramuntanal — COMMENTED** [review]
Good @spuigmitjadxc !
Just some details to review

**Review by Leusev — CHANGES_REQUESTED** [review]
Looks good for me @spuigmitjadxc 
Only a couple of things:
1. Could you check with rubocop lint test please?
image

2. When PR #7 is reviewed/merged with master, then you'll could merge this PR from master in order to automatically fix most PR errors

**Comment by tramuntanal** [comment]
PR #7 has been merged

**Review by Leusev — CHANGES_REQUESTED** [review]
We already have merged PRs #11  #13 and  #15, do master merge please :smile:

**Comment by Leusev** [comment]
Hi @spuigmitjadxc 
Could you check Lint tests please?
Thanks!

**Comment by tramuntanal** [comment]
Hi @spuigmitjadxc I've executed the tests locally and 9 out of 26 are failing:
imatge

Please make sure all tests succeed

**Comment by spuigmitjadxc** [comment]
PR migred to #17 cause 14 commits aren't from this branch

**Pull request closed without merging**